//
// Academic License - for use in teaching, academic research, and meeting
// course requirements at degree granting institutions only.  Not for
// government, commercial, or other organizational use.
//
// acdefg_NO_VP.h
//
// Code generation for function 'acdefg_NO_VP'
//

#ifndef ACDEFG_NO_VP_H
#define ACDEFG_NO_VP_H

// Include files
#include "rtwtypes.h"
#include <cstddef>
#include <cstdlib>

// Function Declarations
void acdefg_NO_VP(double P_init, double V_init, double A_init, double P_wayp,
                  double V_wayp, double V_max, double V_min, double A_min,
                  double J_max, double J_min, creal_T t[28]);

#endif
// End of code generation (acdefg_NO_VP.h)
